.github/ISSUE_TEMPLATE/02_bug.yaml

```diff
@@ -1,6 +1,6 @@
 name: "🐛 Bug Report"
 description: Describe a bug in the parcels codebase. For support related to your specific use case, please post in Q&A in our Discussions tab.
-labels: ["bug"]
+labels: ["bug", "needs-triage"]
```

Comment:
Do we need to explain what we mean with "triage"? Not sure all our users would know the concept.